auth_signup_verify_email/controllers/main.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2015 Antiun Ingeniería, S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import logging
+from email_validator import validate_email, EmailSyntaxError, \
+    EmailUndeliverableError
+from odoo import _
+from odoo.http import request, route
+from odoo.addons.auth_signup.controllers.main import AuthSignupHome
+
+_logger = logging.getLogger(__name__)
+
+
+class SignupVerifyEmail(AuthSignupHome):
+
+    # HACK to prevent crashes if other modules modify "qcontext" in wrong
```

Comment:
```suggestion
    # HACK https://github.com/odoo/odoo/pull/34690
```

And add it to roadmap please.